spring-cloud-contract-wiremock/src/main/java/org/springframework/cloud/contract/wiremock/AutoConfigureWireMock.java

```diff
@@ -40,7 +41,7 @@
 @Retention(RetentionPolicy.RUNTIME)
 @Documented
 @Import(WireMockConfiguration.class)
-@PropertyMapping("wiremock.server")
+@PropertyMapping(value = "wiremock.server", skip = SkipPropertyMapping.ON_DEFAULT_VALUE)
```

Comment:
This is the fix, by comparing/contrasting w/ our tried-true friend [AutoConfigureMockMvc](https://github.com/spring-projects/spring-boot/blob/master/spring-boot-project/spring-boot-test-autoconfigure/src/main/java/org/springframework/boot/test/autoconfigure/web/servlet/AutoConfigureMockMvc.java#L63) I was able to see the magic that we needed to apply here.